bitbots_quintic_walk/src/QuinticWalkingNode.cpp

```diff
@@ -230,10 +230,14 @@ void QuinticWalkingNode::imuCb(const sensor_msgs::Imu msg) {
         // compute the pitch offset to the currently wanted pitch of the engine
         double wanted_pitch = _params.trunkPitch + _params.trunkPitchPCoefForward*_walkEngine.getFootstep().getNext().x()
         + _params.trunkPitchPCoefTurn*fabs(_walkEngine.getFootstep().getNext().z());
-
         pitch = pitch - wanted_pitch;
 
-        if (abs(roll) > _imu_roll_threshold || abs(pitch) > _imu_pitch_threshold) {
+        // get angular velocities
+        double roll_vel = msg.angular_velocity.x;
+        double pitch_vel = msg.angular_velocity.y;
+
+
+        if (abs(roll) > _imu_roll_threshold || abs(pitch) > _imu_pitch_threshold || pitch_vel > _imu_pitch_vel_threshold || roll_vel > _imu_roll_vel_threshold) {
```

Comment:
I think that the absolute value of the velocities should be used

Reply:
good point. I corrected it :)